src/Transpiler.ts

```diff
@@ -1286,6 +1286,9 @@ export abstract class LuaTranspiler {
         return result;
     }
 
+    // Implemented in 5.1 and overridden in 5.2 (and onwards)
+    public abstract transpileVariableDestructuring(value: string): string;
```

Comment:
By convention we throw a TranspileError here, see `transpileContinue` in Transpiler.ts.